docs/getting-started/installation/plesk.md

```diff
@@ -125,6 +125,15 @@ Afterwards try `bootstrap` again.
 
 :::
 
+## 8. Clean up .env file
+
+It is good security practice not to leave emails and passwords stored in plain text, so if you had to create login details through the .env file you should no remove those details from the file.
+
+## 9. Test Directus Access
+
+The Directus app should now work under your configured url. If not, try changing the development mode and wait a couple
```

Comment:
I've just noticed that I've used the wrong term when I wrote the guide initially. While you're at it, it would be good to change "development mode" to the term "application mode". Thanks :)